components/net/connector.rs

```diff
@@ -29,7 +29,9 @@ const DEFAULT_CIPHERS: &'static str = concat!(
 
 pub fn create_http_connector() -> Arc<Pool<Connector>> {
     let mut context = SslContext::new(SslMethod::Sslv23).unwrap();
-    context.set_CA_file(&resources_dir_path().join("certs")).unwrap();
+    context.set_CA_file(&resources_dir_path()
+                        .expect("Need certificate file to make network requests")
```

Comment:
We could fall back to an HTTP-only mode here if we wanted, but that's not too important for now.